test/common/protobuf/utility_test.cc

```diff
@@ -174,10 +174,10 @@ TEST_F(ProtobufUtilityTest, DowncastAndValidateUnknownFields) {
   envoy::config::bootstrap::v3alpha::Bootstrap bootstrap;
   bootstrap.GetReflection()->MutableUnknownFields(&bootstrap)->AddVarint(1, 0);
   EXPECT_THROW_WITH_MESSAGE(TestUtility::validate(bootstrap), EnvoyException,
-                            "Protobuf message (type envoy.config.bootstrap.v2.Bootstrap with "
+                            "Protobuf message (type envoy.config.bootstrap.v3alpha.Bootstrap with "
```

Comment:
Probably not actionable in this PR, but for future, do you think we should add a macro for these types in string, or do wild card matching like `envoy.*.Bootstrap`?

Reply:
I'll take this one as a followup, I agree it's a nice to have.